### ISSUE TITLE
1.18.2 Crashes with Continuity

### ISSUE THREAD

**Issue by mrdhobbs** [PepperCode1/Continuity#62]
I have just upgraded Minecraft to 1.18.2 (along with all updated mods), and when loading a world, it gets to 100%, then the world is saved and Minecraft crashes. Here is the report:

https://paste.ee/p/OMzwS

The main errors appear to be

`[Worker-Main-7/ERROR]: Using missing texture, unable to load minecraft:textures/continuity_reserved/cde700970e71c6c5cbf79b4e2a33ace4.png : java.io.FileNotFoundException: minecraft:textures/continuity_reserved/cde700970e71c6c5cbf79b4e2a33ace4.png`

and

`[Render thread/ERROR]: Unreported exception thrown!
java.lang.IncompatibleClassChangeError: Found interface net.minecraft.class_5455, but class was expected
	at me.pepperbell.continuity.client.util.biome.BiomeHolderManager.refreshHolders(BiomeHolderManager.java:41) ~[continuity-1.1.0+1.18.jar:?]...`

Please advise

**Comment by Julienraptor01** [comment]
I think the mod will need to be updated to function on 1.18.2. So please wait until the update.

**Comment by maltreuter** [comment]
** edit: running fabric, sodium, indium (all updated to latest 1.18.2 versions)
I have a similar error here: 

```
Time: 3/5/22, 1:11 PM
Description: Unexpected error

java.lang.IncompatibleClassChangeError: Found interface net.minecraft.class_5455, but class was expected
	at me.pepperbell.continuity.client.util.biome.BiomeHolderManager.refreshHolders(BiomeHolderManager.java:41)
	at me.pepperbell.continuity.client.util.biome.BiomeHolderManager.setup(BiomeHolderManager.java:32)
	at me.pepperbell.continuity.client.handler.ClientPlayJoinHandler.onPlayReady(ClientPlayJoinHandler.java:12)
	at net.fabricmc.fabric.api.client.networking.v1.ClientPlayConnectionEvents.lambda$static$2(ClientPlayConnectionEvents.java:53)
	at net.fabricmc.fabric.impl.networking.client.ClientPlayNetworkAddon.onServerReady(ClientPlayNetworkAddon.java:67)
	at net.minecraft.class_634.handler$bbi000$handleServerPlayReady(class_634.java:2927)
	at net.minecraft.class_634.method_11120(class_634.java:395)
	at net.minecraft.class_2678.method_11567(class_2678.java:83)
	at net.minecraft.class_2678.method_11054(class_2678.java:19)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:157)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:131)
	at net.minecraft.class_1255.method_5383(class_1255.java:116)
	at net.minecraft.class_310.method_1523(class_310.java:1085)
	at net.minecraft.class_310.method_1514(class_310.java:737)
	at net.minecraft.client.main.Main.main(Main.java:236)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at me.pepperbell.continuity.client.util.biome.BiomeHolderManager.refreshHolders(BiomeHolderManager.java:41)
	at me.pepperbell.continuity.client.util.biome.BiomeHolderManager.setup(BiomeHolderManager.java:32)
	at me.pepperbell.continuity.client.handler.ClientPlayJoinHandler.onPlayReady(ClientPlayJoinHandler.java:12)
	at net.fabricmc.fabric.api.client.networking.v1.ClientPlayConnectionEvents.lambda$static$2(ClientPlayConnectionEvents.java:53)
	at net.fabricmc.fabric.impl.networking.client.ClientPlayNetworkAddon.onServerReady(ClientPlayNetworkAddon.java:67)
	at net.minecraft.class_634.handler$bbi000$handleServerPlayReady(class_634.java:2927)
	at net.minecraft.class_634.method_11120(class_634.java:395)
	at net.minecraft.class_2678.method_11567(class_2678.java:83)
	at net.minecraft.class_2678.method_11054(class_2678.java:19)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:157)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:131)
```

**Comment by PepperCode1** [comment]
Resolved with commit 7cdd12331c2df058eaac8f24b61e2b8931c498a3. Releases for 1.18.2 have been published on CurseForge, Modrinth, and GitHub Releases as usual.
